.github/workflows/label.yml

```diff
@@ -1,24 +0,0 @@
-# This workflow will triage pull requests and apply a label based on the
-# paths that are modified in the pull request.
-#
-# To use this workflow, you will need to set up a .github/labeler.yml
-# file with configuration.  For more information, see:
-# https://github.com/actions/labeler
-
-name: Labeler
-on: [pull_request_target]
-
-jobs:
-  label:
-    runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      pull-requests: write
-
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v4
-      - name: Add label
-        uses: actions/labeler@v5
-        with:
-          repo-token: '${{ secrets.GITHUB_TOKEN }}'
```

Comment:
I think this might've been deleted inadvertently post-merge; this should still appear/exist, based on current `main`